modules/afmongodb/afmongodb.c

```diff
@@ -142,26 +146,53 @@ afmongodb_dd_disconnect(LogThrDestDriver *s)
 }
 
 static gboolean
-afmongodb_dd_connect(MongoDBDestDriver *self, gboolean reconnect)
+_dd_connect(MongoDBDestDriver *self, gboolean reconnect)
 {
```

Comment:
I like the shortened names, that's really great.

The DD part of the name here is probably not needed though. That refers to destination driver, and none of the other functions retained those in the name, and is probably not needed here either